AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLModelBased/README.md

```diff
@@ -0,0 +1,44 @@
+## Model Based GraphQL 
+
+The following steps show how ot set up a GraphQL endpoint with AppSync. The auth configured will be API key. The set up is used to run the tests in `GraphQLModelBasedTests.swift`
+
+
+### Set-up
+
+1.  `amplify-init`
```

Comment:
add single space after period (or remove a space after `2.` and `3.` to be consistent)

Reply:
updating